src/main.rs

```diff
@@ -109,7 +115,15 @@ fn main() {
 #[allow(unused_mut)] // TODO: Remove when fixed in chan_select
 fn run(matches: &ArgMatches) -> Result<()> {
     // Now we can start to run the i3bar protocol
-    print!("{{\"version\": 1, \"click_events\": true}}\n[");
+    let initialise = if matches.is_present("never-pause") {
+        format!(
+            "{{\"version\": 1, \"click_events\": true, \"stop_signal\": {}}}\n[",
+            nix::sys::signal::Signal::SIGUSR2 as i8
+        )
+    } else {
+        "{{\"version\": 1, \"click_events\": true}}\n[".to_string()
```

Comment:
```suggestion
        "{\"version\": 1, \"click_events\": true}\n[".to_string()
```
I think you don't need to escape `{` here

Reply:
It chucks a sad if you don't escape it, but then it somehow ends up printing both... no idea why but I changed the code around to avoid that.